source/chapters/user_interface.rst

```diff
@@ -370,7 +370,7 @@ buttons depends on the selected Cue mode. To change the Cue mode, go to
 .. seealso:: You can also use keyboard shortcuts for Cueing. Go to
              :ref:`control-keyboard` for more information.
 
-.. _interface-transport:
+_interface-transport:
```

Comment:
Please undo this, I think this was probably changed by accident.

Reply:
Oh, i'm sorry about this. Change undone.